diff -Naur NVIDIA-Linux-x86_64-460.32.03-grid/kernel/conftest.sh NVIDIA-Linux-x86_64-460.32.03-grid-new/kernel/conftest.sh
--- NVIDIA-Linux-x86_64-460.32.03-grid/kernel/conftest.sh	2020-12-27 11:31:38.000000000 -0800
+++ NVIDIA-Linux-x86_64-460.32.03-grid-new/kernel/conftest.sh	2021-04-16 20:45:13.893946912 -0700
@@ -24,6 +24,7 @@
 
 # VGX_BUILD parameter defined only for VGX builds (vGPU Host driver)
 # VGX_KVM_BUILD parameter defined only vGPU builds on KVM hypervisor
+VGX_KVM_BUILD=1
 # GRID_BUILD parameter defined only for GRID builds (GRID Guest driver)
 GRID_BUILD=1
 # GRID_BUILD_CSP parameter defined only for GRID CSP builds (GRID Guest driver for CSPs)
@@ -4334,6 +4335,7 @@
             # in v3.7-rc1 (2012-09-26)
             #
             CODE="
+            #include <linux/types.h>
             #include <linux/fdtable.h>
             void conftest_iterate_fd(void) {
                  iterate_fd();
