diff --git NVIDIA-Linux-x86_64-470.63-vgpu-kvm/kernel/nvidia/nvidia.Kbuild NVIDIA-Linux-x86_64-470.63-vgpu-kvm-new/kernel/nvidia/nvidia.Kbuild
index 7d7751e..9849bd5 100644
--- NVIDIA-Linux-x86_64-470.63-vgpu-kvm/kernel/nvidia/nvidia.Kbuild
+++ NVIDIA-Linux-x86_64-470.63-vgpu-kvm-new/kernel/nvidia/nvidia.Kbuild
@@ -221,3 +221,5 @@ NV_CONFTEST_GENERIC_COMPILE_TESTS += pm_runtime_available
 NV_CONFTEST_GENERIC_COMPILE_TESTS += vm_fault_t
 NV_CONFTEST_GENERIC_COMPILE_TESTS += pci_class_multimedia_hd_audio
 NV_CONFTEST_GENERIC_COMPILE_TESTS += drm_available
+
+ldflags-y += -T @vgpu_unlock@/kern.ld
diff --git NVIDIA-Linux-x86_64-470.63-vgpu-kvm/kernel/nvidia/os-interface.c NVIDIA-Linux-x86_64-470.63-vgpu-kvm-new/kernel/nvidia/os-interface.c
index d04875a..621012a 100644
--- NVIDIA-Linux-x86_64-470.63-vgpu-kvm/kernel/nvidia/os-interface.c
+++ NVIDIA-Linux-x86_64-470.63-vgpu-kvm-new/kernel/nvidia/os-interface.c
@@ -15,6 +15,7 @@
 
 #include "nv-time.h"
 
+#include "@vgpu_unlock@/vgpu_unlock_hooks.c"
 
 
 
