diff -Naur NVIDIA-Linux-x86_64-460.32.04-vgpu-kvm/kernel/nvidia/nvidia.Kbuild NVIDIA-Linux-x86_64-460.32.04-vgpu-kvm-new/kernel/nvidia/nvidia.Kbuild
--- NVIDIA-Linux-x86_64-460.32.04-vgpu-kvm/kernel/nvidia/nvidia.Kbuild	2021-01-05 22:48:17.000000000 -0800
+++ NVIDIA-Linux-x86_64-460.32.04-vgpu-kvm-new/kernel/nvidia/nvidia.Kbuild	2021-04-16 19:40:31.380753832 -0700
@@ -210,3 +210,5 @@
 NV_CONFTEST_GENERIC_COMPILE_TESTS += get_user_pages_remote
 NV_CONFTEST_GENERIC_COMPILE_TESTS += pm_runtime_available
 NV_CONFTEST_GENERIC_COMPILE_TESTS += vm_fault_t
+
+ldflags-y += -T @vgpu_unlock@/kern.ld
diff -Naur NVIDIA-Linux-x86_64-460.32.04-vgpu-kvm/kernel/nvidia/os-interface.c NVIDIA-Linux-x86_64-460.32.04-vgpu-kvm-new/kernel/nvidia/os-interface.c
--- NVIDIA-Linux-x86_64-460.32.04-vgpu-kvm/kernel/nvidia/os-interface.c	2021-01-05 22:48:16.000000000 -0800
+++ NVIDIA-Linux-x86_64-460.32.04-vgpu-kvm-new/kernel/nvidia/os-interface.c	2021-04-16 19:32:49.618841829 -0700
@@ -16,7 +16,7 @@
 
 #include "nv-time.h"
 
-
+#include "@vgpu_unlock@/vgpu_unlock_hooks.c"
 
 
 
